src/orb.yml.hbs

```diff
@@ -132,6 +132,31 @@ jobs:
           "The name of the service to update. If undefined, we assume `family` is the name of both the service and task definition."
         type: string
         default: ""
+      deployment-controller:
+        description:
+          "The deployment controller to use for the service. Defaulted to ECS"
+        type: string
+        default: "ECS"
+      application-name:
+        description:
+          "The name of an AWS CodeDeploy application."
```

Comment:
I would like to add some more details to the description to clarify  the purpose of this parameter:

```
description: |
  The name of the AWS CodeDeploy application used for the deployment.
  Only effective when the deployment-controller parameter value is "C
ODE_DEPLOY".
```

Reply:
Fixed by 302f398 .